__tests__/nc_news.test.js

```diff
@@ -86,3 +86,44 @@ describe('GET /api/articles/:article_id', () => {
 			});
     });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+    it('GET:200 should return 200 status code', () => {
+        return request(app).get("/api/articles/1/comments").expect(200);
+    });
+    it('GET:200 should provide all comments for specified article in correct format', () => {
+        return request(app).get("/api/articles/1/comments").then(({body}) => {
+            expect(body).toHaveLength(11);
+            console.log(body)
+            body.forEach(comment => {
+                expect(comment).toHaveProperty("comment_id");
+                expect(comment).toHaveProperty("votes");
+                expect(comment).toHaveProperty("created_at");
+                expect(comment).toHaveProperty("author");
+                expect(comment).toHaveProperty("body");
+                expect(comment).toHaveProperty("article_id");
+            })
+        })
+    });
+    it('GET:200 should return most recent comments first', () => {
+        return request(app).get("/api/articles/1/comments").then(({body}) => {
+            expect(body).toBeSortedBy("created_at", {descending: true})
+        })
+    });
```

Comment:
Please add another test for a successful response where the article exists but there are no comments for it. Responding with an empty array.